bindata/network/ovn-kubernetes/ovnkube-master.yaml

```diff
@@ -115,6 +115,25 @@ spec:
 
           bracketify() { case "$1" in *:*) echo "[$1]" ;; *) echo "$1" ;; esac }
 
+          check_db_health() {
+            db=$1
```

Comment:
ah, well technically we should quote here as well

`db="${1}"`

Reply:
we should probably run these snippets through shellcheck regularly.

https://www.shellcheck.net/

Reply:
Done